evm/logic/logging.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import
-
+from typing import List  # noqa: F401
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically